app/assets/javascripts/components/overview/available_actions.jsx

```diff
@@ -100,7 +101,8 @@ const AvailableActions = createReactClass({
     const course = this.state.course;
     const onConfirm = function () {
       enableRequests(course);
-      return alert(I18n.t('courses.accounts_generation_enabled'));
+      alert(I18n.t('courses.accounts_generation_enabled'));
+      return CourseActions.updateCourse(course);
```

Comment:
@ragesoss this is not working as I expected. It's not rerendering the component so the button is still appearing until the user refresh the page. Do you have any idea why?